shared/Microsoft.AspNetCore.Certificates.Generation.Sources/CertificateManager.cs

```diff
@@ -33,9 +33,13 @@ internal class CertificateManager
         public const int RSAMinimumKeySizeInBits = 2048;
 
         private static readonly TimeSpan MaxRegexTimeout = TimeSpan.FromMinutes(1);
-        private const string MacOSFindCertificateCommandLine = "security";
-        private const string MacOSFindCertificateCommandLineArguments = "find-certificate -c localhost -a -Z -p /Library/Keychains/System.keychain";
+        private const string MacOSSystemKeyChain = "/Library/Keychains/System.keychain";
+        private static readonly string MacOSUserKeyChain = Environment.GetEnvironmentVariable("HOME") + "/Library/Keychains/login.keychain-db";
```

Comment:
How will we ever (in the future) handle a situation where the key chain name or location changes in the future versions of MacOS?

Reply:
That will be a huge all breaking changes in OSX, but in any case, we would update the tool at that point.

Reply:
Ok